src/features/operator/operator.api.ts

```diff
@@ -836,14 +848,53 @@ export const operatorApi = tdexApi.injectEndpoints({
                 }))
             )
           );
+          console.log({ results, assets });
           for (const r of results) {
             // eslint-disable-next-line
             r.market.forEach((marketId) => {
               const marketIdCollectedSwapFees = r.getMarketCollectedSwapFeesReply
                 .getTotalCollectedFeesPerAssetMap()
                 .get(marketId);
-              if (marketIdCollectedSwapFees) {
-                totalCollectedSwapFees += marketIdCollectedSwapFees;
+
+              // Get Asset Data using the asset marketId
+              const currentAsset = getAssetDataFromRegistry(marketId, assets[network], 'L-BTC');
+              console.log({ currentAsset });
+
+              if (currentAsset === undefined || marketIdCollectedSwapFees === undefined) {
+                // No point in calculating asset conversion
+                // for unknown asset and
+                // for undefined marketIdCollectedSwapFees
+                return;
+              }
+
+              console.log({ marketIdCollectedSwapFees });
+
+              // Keeps marketIdCollectedSwapFees as sats if it's btc
+              // 100000000 lbtc/sats-representation = 100000000 L-sats
+              // Otherwise this makes sure to get the amount with fiat precision
+              // 100000000 usdt/sats-representation = 1 usdt
+              const currentAmount = fromSatsToUnitOrFractional(
+                marketIdCollectedSwapFees,
+                currentAsset.precision,
+                isLbtcTicker(currentAsset.ticker),
+                'L-sats'
+              );
+              console.log({ currentAmount });
+
+              // Converts (LBTC/USDT/LCAD) to L-sats terms using the latest prices
+              const marketIdCollectedSwapFeesInSats = convertAmountToFavoriteCurrency({
+                asset: currentAsset,
+                amount: currentAmount,
+                network: network,
+                preferredCurrency: BTC_CURRENCY,
```

Comment:
`BTC_CURRENCY` should not be hardcoded here. We should display swap fees in any selected favorite currency.

Reply:
I thought it was cleaner for **totalCollectedSwapFees** to return the data in satoshis. That way we have a clear separation of concerns.

Also I got that idea from the [tdex-daemon code 
](https://github.com/tdex-network/tdex-daemon/blob/b5e6c358b6a19419779e5741fb9ee92066cf1539/api-spec/protobuf/gen/go/tdex-daemon/v1/operator.pb.go#L4903=). The daemon specifies "Fees" in Satoshi units.

```js
// returns satoshis
totalCollectedSwapFees = useTotalCollectedSwapFeesQuery(...);

// display fees in favorite currency
totalCollectedSwapFeesAsFavoriteCurrency = convertAmountToFavoriteCurrency({amount: totalCollectedSwapFees, favoriteCurrency: userFavoriteCurrency})

```

Reply:
Yes ok you right, it's better like that.